augly/image/transforms.py

```diff
@@ -1310,7 +1309,36 @@ def apply_transform(
             image, factor=self.factor, seed=self.seed, metadata=metadata
         )
 
-
+    
+class Skew(BaseTransform):
+    def __init__(self, level: float = 0.5, axis: str = "x-axis", p: float = 1.0):
+        """
+        @param level: the level of skew to apply to the image; a larger value means the skew
+            will be more intense. Recommended range is between [-2, 2]
```

Comment:
```suggestion
        @param level: the level of skew to apply to the image; a larger absolute value will
            result in a more intense skew. Recommended range is between [-2, 2]
```
to match with functional.py